test/unit/helpers/xhr/PASSWORD_RESET_withComplexity.js

```diff
@@ -8,14 +8,14 @@ define({
     "_embedded": {
       "policy": {
         "age": {
-          "historyCount": 0
+          "historyCount": -1
```

Comment:
if I remember correctly this values don't need to change, but instead we need to add a test in ```PasswerReset_spec``` for -1 (that spec overrides the values from this xhr mock)